### PR TITLE
Findmax

### PR DESCRIPTION
**Pull request by janiekyu**

### CODE REVIEW

**Comment by janiekyu** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401challenges/src/test/java/code401challenges/tree/TreeFindMax.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        Node leftLeftNode = new Node(5);
+        Node leftrightNode = new Node(6);
+        Node rightrightNode = new Node(25);
+        testTree.root = rootNode;
+        rootNode.leftChild = leftNode;
+        rootNode.rightChild = rightNode;
+        leftNode.leftChild = leftLeftNode;
+        leftNode.rightChild = leftrightNode;
+        rightNode.rightChild = rightrightNode;
+
```
Comment (Tw1stedRain) [inline code401challenges/src/test/java/code401challenges/tree/TreeFindMax.java]: this test has no asserts
